### PR TITLE
V2

### PR DESCRIPTION
**Pull request by deathiop**
To merge after #114 and #113

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -426,6 +414,32 @@ Example usage:
...
+    # Call to https://eu.api.ovh.com/v1/xdsl/xdsl-yourservice
+    client.get("/v1/xdsl/xdsl-yourservice")
+
+    # Call to https://eu.api.ovh.com/v2/xdsl/xdsl-yourservice
+    client.get("/v2/xdsl/xdsl-yourservice")
+
+    # Legacy call to https://eu.api.ovh.com/1.0/xdsl/xdsl-yourservice
+    client.get("/xdsl/xdsl-yourservice")
+
+
```
Comment (amstuta) [inline README.rst:442]:
```suggestion
```

File: ovh/client.py [hunk 2]
```diff
@@ -476,6 +472,20 @@ def call(self, method, path, data=None, need_auth=True):
         else:
             raise APIError(json_result.get("message"), response=result)
 
+    def _get_target(self, path):
+        """
+        getTarget returns the URL to target given and endpoint and a path.
```
Comment (amstuta) [inline ovh/client.py:477]:
```suggestion
        _get_target returns the URL to target given and endpoint and a path.
```

**Review by amstuta — APPROVED** [review]

File: README.rst [hunk 3]
```diff
@@ -414,6 +413,31 @@ Example usage:
...
+- the legacy URL is https://eu.api.ovh.com/1.0
+
+ Calling ``client.get``, you can target the API version you want:
+
+.. code:: python
+
+    client = ovh.Client(endpoint="ovh-eu")
+
+    # Call to https://eu.api.ovh.com/v1/xdsl/xdsl-yourservice
+    client.get("/v1/xdsl/xdsl-yourservice")
```
Comment (a-beudin) [inline README.rst:433]: Seen together, this call raises an `ovh.exceptions.BadParametersError: Invalid signature` error.

Comment (deathiop) [reply README.rst:433]: Seen together, linked to internal infrastructure, not to this PR.

**Review by a-beudin — DISMISSED** [review]

**Review by rbeuque74 — APPROVED** [review]

**Pull request merged**